.eslintrc.json

```diff
@@ -0,0 +1,71 @@
+{
+  "root": true,
+  "extends": "airbnb",
+  "env": {
+    "node": true,
+    "es6": true,
+    "mocha": true,
+    "browser": true,
+    "jquery": true
+  },
+  "globals": {
+    "Materialize": true
+  },
+  "rules": {
+    "one-var": 0,
```

Comment:
I also remove the edit mode entirely to ensure great user experience. The first admin cannot be deleted and the search bar is now very conspicuous. Thanks a lot for your feedback.